charts/falco/Chart.yaml

```
@@ -19,10 +19,14 @@ maintainers:
     email: cncf-falco-dev@lists.cncf.io
 dependencies:
   - name: falcosidekick
-    version: "0.9.*"
+    version: "0.8.*"
```

```suggestion
    version: "0.9.*"
```

Why are we downgrounding this? I guess it's a leftover, isn't it? :thinking:

---

Yes all of them were left overs, I was trying to figure why the test wasn't passing and completely underlooked this. Thanks Leo, I'll fix them and the docs.